testdata/go/v3/monitoring/memcached-operator/docs/monitoring/metrics.md

```diff
@@ -0,0 +1,11 @@
+# Memcached Operator Metrics
+This document aims to help users that are not familiar with metrics exposed by the memcached-operator.
+All metrics documented here are auto-generated by the utility tool "monitoring/metricsdocs" and reflects exactly what is being exposed.
```

Comment:
Just to elaborate, do we need to add that its the responsibility of the operator author to make sure the metrics are defined with the apt metric definition.

Reply:
we did mentioned it in this doc:
"
After developing new metrics or changing old ones, please run "make generate-metricsdocs" to regenerate this document.
"
do you think of another place that might fit?

Reply:
Not sure. This document is mainly to list the metrics available.
@sradco what do you think?

Reply:
I think we can write that the metrics description will be shown in an auto generated metrics documentation and provide a link to the best practices document for additional details.